common/src/main/kotlin/no/nav/su/se/bakover/common/Config.kt

```diff
@@ -543,5 +563,30 @@ data class ApplicationConfig(
                     else -> PREPROD
                 }
         }
+
+        sealed class Konsistensavstemming {
+            abstract val kjøreplan: Set<LocalDate>
+
+            data class Prod(
+                override val kjøreplan: Set<LocalDate> = setOf(
+                    22.november(2021),
```

Comment:
Kan muligens fjerne denne? Vi kan forøvrig sikkert vurdere om disse burde ligge i env eller database eller noe på et senere tidspunkt.